solang-parser/src/solidity.lalrpop

```diff
@@ -121,6 +121,15 @@ StorageLocation: StorageLocation = {
     <l:@L> "calldata" <r:@R> => StorageLocation::Calldata(Loc::File(file_no, l, r)),
 }
 
+IdentifierProject: Vec<Identifier> = {
+    <i:SolIdentifier?> => vec![],
+    <i:SolIdentifier> <v:("." SolIdentifier)*> => {
```

Comment:
This is ambiguous: both productions above can produce a single `SolIdentifier`. To fix this:

```
IdentifierProject: Vec<Identifier> = {
    => vec![],
    <i:SolIdentifier> <v:("." SolIdentifier)*> => {
            let mut i = vec![i];
            i.extend(v);
            i
    }
}
```

Reply:
great, that worked, updated.